keps/1834-copy-labels-into-workload/README.md

```diff
@@ -0,0 +1,103 @@
+# KEP-1834: Copy some labels from Pods/Jobs into the Workload object 
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [Risks and Mitigations](#risks-and-mitigations)
+- [Design Details](#design-details)
+  - [Test Plan](#test-plan)
+      - [Prerequisite testing updates](#prerequisite-testing-updates)
+    - [Unit Tests](#unit-tests)
+    - [Integration tests](#integration-tests)
+  - [Graduation Criteria](#graduation-criteria)
+- [Implementation History](#implementation-history)
+- [Drawbacks](#drawbacks)
+- [Alternatives](#alternatives)
+<!-- /toc -->
+
+## Summary
+
+When creating a kueue workload, copy the job (or pod) labels into the workload object (see https://github.com/kubernetes-sigs/kueue/issues/1834). Allow to configure which labels should be copied.
+
+## Motivation
+
+Currently the workloads do not "inherit" any of the labels of the jobs based on which they are created. Since workloads in Kueue are internal representations of the Kubernetes jobs, allowing to easily copy the labels of the job object into the workload object is natural and useful. For instance it would facilitate selection/filtering of the workloads by the administrators. 
+
+### Goals
```

Comment:
What about labels updated for an existing Job? Should we update it , or keep the original only? If we keep the original add it to non-goals.

Reply:
I think the idea is to copy the labels only at the moment of creation of the workload. Updating is out of scope (please correct me if I'm wrong). 

Added to non-goals.

Reply:
@pajakd So, what happens when the users update the label of Job? Will we implement validations to prevent updating job labels?

Reply:
sgtm, thanks for updating the KEP

Reply:
Yes, nothing happens if the user changes the labels. I added this information to the API documentation